WhaleSpotting/Models/Enums/Species.cs

```diff
@@ -0,0 +1,22 @@
+﻿namespace WhaleSpotting.Models.Enums
+{
+    public enum Species
+    {
+        Orca, 
```

Comment:
Could we order these and the other enums alphabetically, and also for the first value in each enum put an `= 1`

Reply:
comment addressed.